src/test/java/net/sf/jabref/importer/fileformat/MedlinePlainImporterTest.java

```diff
@@ -201,11 +204,27 @@ public void testWithNbibFile() throws IOException, URISyntaxException {
     public void testWithMultipleEntries() throws IOException, URISyntaxException {
         Path file = Paths
                 .get(MedlinePlainImporter.class.getResource("MedlinePlainImporterStringOutOfBounds.txt").toURI());
-        List<BibEntry> entries = importer.importDatabase(file, Charset.defaultCharset()).getDatabase().getEntries();
+        List<BibEntry> entries = importer.importDatabase(file, Charsets.UTF_8).getDatabase().getEntries();
         BibEntryAssert.assertEquals(MedlinePlainImporter.class, "MedlinePlainImporterStringOutOfBounds.bib", entries);
     }
 
     @Test
+    public void testInvalidFormat() throws URISyntaxException, IOException {
+        Path file = Paths
+                .get(MedlinePlainImporter.class.getResource("MedlinePlainImporterTestInvalidFormat.xml").toURI());
+        List<BibEntry> entries = importer.importDatabase(file, Charsets.UTF_8).getDatabase().getEntries();
+        assertEquals(Collections.EMPTY_LIST, entries);
```

Comment:
Use the method Collections,emptyList() rather than the constant, because the constant does not return a generic list and is therefore not typesafe.
(In this case not so important, but could possinly produce an unchecked warning)